app/src/main/res/layout/activity_chart.xml

```diff
@@ -20,19 +20,12 @@
         app:srcCompat="@drawable/ic_exit_full_screen"
         app:fabSize="normal" />
 
-    <LinearLayout
-        android:id="@+id/progressView"
+    <RelativeLayout
```

Comment:
This can be FrameLayout. No need of RelativeLayout here, RelativeLayout is expensive